aries/aries_vcx/tests/utils/test_agent.rs

```diff
@@ -103,15 +103,14 @@ where
     W: BaseWallet,
 {
     let acme = create_test_agent(genesis_file_path.to_string()).await;
-    let acme_vk =
-        get_verkey_from_wallet(acme.wallet.get_wallet_handle(), &acme.institution_did).await?;
+    let acme_vk = acme.wallet.did_key(&acme.institution_did).await?;
 
     write_endorser_did(
         &trustee_wallet,
         &ledger_write,
         trustee_did,
         &acme.institution_did,
-        &acme_vk,
+        &acme_vk.base58(),
```

Comment:
Improvement but generally not a huge fan of the current state either. This comment is not actionable, just noting for awareness or to spark any discussion. But this shouldn't block this PR.

- Arguably `write_endorser_did` would also take a typed key and there wouldn't be to run the encoding at this spot, just passing around raw data at higher levels of code (like here)
- Arguably we would have types for specific keys, eg. currently there's silent contract that the `target_vk: &str,` argument of `write_endorser_did` in expected ed25519 public key. The `Key` is very general type, as it can represent any key. So ideally, there would be type such as `PublicKeyEd25519` which could be used here.
- All that being said, I would be reluctant to do relevant changes across the codebase just yet - I suppose `askar` might be coming with its own key types and for me it's yet open question whether we'd use those, or abstract them out with custom type.